mitthesis.cls

```diff
@@ -441,8 +436,7 @@ reserved.\par\permission}}
 \def\supervisor#1#2{\setbox\@titlesupervisor\vbox
   {\unvbox\@titlesupervisor \vskip 10pt% plus 1fil minus 1fil
   \def\baselinestretch{1}\large
-  \noindent\choosecase{\small Certified by:}\hspace{1.11cm}\parbox[t]{11cm}{\small #1 \\ #2 \\ Thesis Supervisor}}
-  %\signature{Certified by}{#1 \\ #2 \\ Thesis Supervisor}}
+  \signature{Certified by}{#1 \\ #2 \\ Thesis Supervisor}}
```

Comment:
Should we maintain the use of `\choosecase` here? It was present in the original thesis to allow people to make things uppercase, if they want.

Reply:
This actually wasn't present in the original, I had misremembered (and not sure why I put it there in the earlier version).